lib/matplotlib/tests/test_collections.py

```diff
@@ -325,6 +325,18 @@ def test__EventCollection__set_linestyle():
     splt.set_title('EventCollection: set_linestyle')
 
 
+@image_comparison(baseline_images=['EventCollection_plot__set_ls_dash'])
```

Comment:
In this case we also had a known 'remake all the tests' change to text layout PR in between.